Dockerfile

```diff
@@ -6,12 +6,15 @@ COPY ./pyproject.toml /app
 RUN pip install poetry && poetry config virtualenvs.create false --local && poetry install
 
 FROM base as production
+RUN apt-get update && \
+    apt-get install -y gunicorn
```

Comment:
As you run the app with gunicorn tracked with poetry (i.e. `poetry install` has installed gunicorn) this should be unnecessary - did you see issues without this?

Reply:
removed and tested